src/test/java/org/prebid/server/it/SanityTest.java

```diff
@@ -0,0 +1,23 @@
+package org.prebid.server.it;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.prebid.server.VertxTest;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@TestPropertySource(properties = {
+        "spring.config.additional-location=sample-prebid-config.yaml",
+        "http.port=8081",
```

Comment:
Why not include these values in the config file?

Reply:
Actually, this is dirty hack for JUnit - sanity test starts before previous integration test finished. So, i obtained binding port error. For now i don't have a solution and just override default ports.